packages/next/src/server/lib/server-ipc.ts

```diff
@@ -35,6 +36,9 @@ export async function createIpcServer(
           res.end(JSON.stringify(result || ''))
         }
       } catch (err: any) {
+        if (isError(err) && err.code === 'ENOENT') {
```

Comment:
Should this be inverted e.g. `err.code !== 'ENOENT'` also can we add a test case for this somewhere?